### DISCUSSION TITLE
App for TizenOS

### DISCUSSION THREAD

**Discussion by r00terAF18** [Docile-Alligator/Infinity-For-Reddit#375] [Help needed] [answered]
Hello, my name is Amir and recently I purchased a Watch Active 2, for getting back to C# and learning more I decided that I want to have my reddit feed ( in a much simpler form) on my watch. Since I'm not a designer, very bad at UI and anything related, thought of asking you for permission to use the Icon and name. My app is going to be free, I haven't started really doing anything, rather just messing around with Xamarin, but I was hoping to get some advice before starting out. Thank you very much

**Comment by Wladefant** [comment, marked as answer]
Hi,
Did you start your project?

**Reply by KSPAtlas** [reply]
just so you know, you use the infinitive (start) in this case, so it would be “Did you start your project?”
